wunderio/next4drupal-project
# NEX-81: Update File Share Component

## Link to ticket

[**NEX-81**](https://wunder.atlassian.net/browse/NEX-81)

## Link to feature environment:

[**Test Link**](https://feature-nex-81-next.next4drupal.dev.wdr.io/)

## Changes proposed in this PR:

I added the changes we made in Nemo project to allow for pdf and word/text format files to have associated svgs.

A11y score is 100%, the voiceover has been tested and sounds logical as well as having translations.

## How to test:

1. Log into the backend to add and remove files. 
```
ssh www-admin@feature-nex-81-shell.next4drupal-project -J www-admin@ssh.dev.wdr.io "drush uli"
```
2. Then check the frontend to see that they are rendered. 
```
https://feature-nex-81-next.next4drupal.dev.wdr.io/
```

## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
